snap/info_test.go

```diff
@@ -29,6 +29,7 @@ import (
 	"strings"
 	"testing"
 
+	"gopkg.in/check.v1"
```

Comment:
do we really need this?

Reply:
Fixed... auto-save some times guesses things